.github/workflows/ci.yml

```diff
@@ -402,6 +402,39 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y criu
 
+      - name: Build and install erofs-utils from source
```

Comment:
Could you add a comment string to explain why [`apt-get install erofs-utils`](https://launchpad.net/ubuntu/noble/+source/erofs-utils) can't be used here?

Reply:
The comment still does not explain why 1.7 is not enough

Reply:
The main reason is that 1.7 is not the latest version even though it may be fine to use it.

Please refer to https://github.com/containerd/containerd/pull/11993#discussion_r2162151576, cc @hsiangkao